workflow/rocoto/gfs_tasks.py

```diff
@@ -110,15 +110,18 @@ def waveinit(self):
 
         resources = self.get_resource('waveinit')
         dependencies = None
+        cycledef = None
```

Comment:
What does this do?

Reply:
@aerorahul Was this a commit that I made?

Reply:
seems like it. It shows up as an addition on the review.

Reply:
This was added in the case that the `if self.app_config.mode in ['cycled']:` block is not entered. This was a fail-safe to avoid raising an  exception.

Reply:
In that case `None` is not the right choice.  The default for `cycledef` should be `self.cdump`.

Reply:
I take that back, `None` is fine, because [this](https://github.com/NOAA-EMC/global-workflow/blob/13d06ded4155cba2d08ef2a2cd553024c5abe29d/workflow/rocoto/tasks.py#L199)